services/gitlab/gitlab-merge-requests.service.js

```diff
@@ -290,20 +291,11 @@ export default class GitlabMergeRequests extends GitLabBase {
     return {
       label: `${labelPrefix}${labelText}merge requests`,
       message,
-      color: mergeRequestCount > 0 ? 'brightgreen' : 'yellow',
+      color: mergeRequestCount > 0 ? 'brightgreen' : 'blue',
```

Comment:
Sorry if I was unclear previously, but what I was saying is that the color should _always_ be blue and not having a conditional color.